code/modules/customitems/item_defines.dm

```diff
@@ -326,6 +326,11 @@
 		if(H.head == src)
 			H.slurring = max(3, H.slurring) //always slur
 
+/obj/item/clothing/head/beret/fluff/linda //Epic_Charger: Linda Clark
+	name = "Green beret"
+	desc = "A beret, an artists favorite headwear. This one has two holes cut on the edges."
```

Comment:
I didn't pay attention to the stuff Epic sent me and I just copypasted. It was late, don't judge.